php/seatreg_functions.php

```diff
@@ -593,6 +593,21 @@ function seatreg_generate_settings_form() {
 				<textarea class="form-control" id="registration-info-text" name="registration-info-text" placeholder="<?php esc_html_e('Enter info text here', 'seatreg'); ?>"><?php echo esc_html($options[0]->info); ?></textarea>
 			</div>
 
+			<div class="form-group">
+				<label for="custom-footer-text"><?php esc_html_e('Custom footer text', 'seatreg'); ?></label>
```

Comment:
What do you think about changing the **Custom footer text** label to be something like **Registration custom footer area**?

Reply:
Agree. Changed: https://github.com/SiimKirjanen/seatreg/commit/565febfd0c182fd810bd494c738c934e40fe049b